ChangeLog

```diff
@@ -1,3 +1,30 @@
+# 2021-02-05 Version 2.3.0
+
+Important notes:
+* CMake option WITH_PROXY_MODULES is currently experimental, do not use in
+production.
+
+Noteworthy changes:
+* Progressive codec improvements. Reduces graphical glitches against windows
+and ogon servers
+* Fixed +glyph-cache, now working properly without disconnects
+* Huge file support in clipboard
+* XWayland support for xfreerdp (keyboard grabbing)
+* Imporved wlfreerdp (wayland client)
```

Comment:
@akallabeth small typo: Imporved->Improved

Reply:
@Fantu thank you, missed that one ^^